minecraft.js

```diff
@@ -96,12 +115,33 @@ function shutdownMinecraft() {
 function restartMinecraft() {
   if (serverStatus == MINECRAFT_STATUS.RUNNING) {
     shutdownMinecraft();
-    startMinecraft();
-  } else {
-    startMinecraft();
   }
+  startMinecraft();
 }
 
 function getMinecraftStatus() {
   return serverStatus;
 }
+
+function fileToJSON(input) {
+  var output = {};
+
+  input.split('\n').forEach(function (line) {
+    if (line[0] === '#') {
+      return;
+    }
+
+    let parts = line.split('=');
+
+    if (parts[0] === '') {
```

Comment:
What usecase does this handle?

Reply:
It handles empty lines as well as the end of the file.

Reply:
Oh awesome